src/Microsoft.DocAsCode.Build.JavaScriptReference/AutoMapper/ApiLinkInfoBuildOutputUrlResolver.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Build.JavaScriptReference
+{
+    using System.Web;
+
+    using AutoMapper;
+
+    public class ApiHrefLinkInfoBuildOutputUrlResolver : IValueResolver<LinkInfo, ApiLinkInfoBuildOutput, string>
+    {
+        public string Resolve(LinkInfo source, ApiLinkInfoBuildOutput destination, string destMember, ResolutionContext context)
+        {
+            var href = $"<span><a href=\"{source.LinkId}\">";
```

Comment:
AutoMapper will handle null value, and it will no be passed here

---
In reply to: [93932349](https://github.com/dotnet/docfx/pull/1061#discussion_r93932349) [](ancestors = 93932349)